dev-core

```diff
@@ -1,16 +1,49 @@
 #!/bin/bash
 
-echo "Building mesg/core:local..."
-docker build -t mesg/core:local --build-arg version="local" .
+set -e
+
+DIR=${GOPATH:-$HOME/go}
+VERSION=local
+LDFLAGS="-X 'github.com/mesg-foundation/core/version.Version=$VERSION'"
+
+echo "compile cli and core"
+go build -o ./bin/core -ldflags="$LDFLAGS" core/main.go
```

Comment:
```suggestion
GOOS=linux GOARCH=amd64 go build -o ./bin/core -ldflags="$LDFLAGS" core/main.go
```

Reply:
It doesn't work on mac because it's compiled in the host machine. The core needs to be compiled for the container.

Reply:
Error on OSX
```
standard_init_linux.go:190: exec user process caused "exec format error"
```

Reply:
Done